### PR TITLE
Add country-history.downloaddataset and update SV

### PR DESCRIPTION
**Pull request by magol**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -2,18 +2,10 @@ root = true
 
 [*]
 indent_style = space
-indent_size = 2
```
Comment (corradio) [inline .editorconfig]: Can you keep the updates minimal? i.e. only end of line? html should be kept at indent_size 4.

File: web/views/pages/index.ejs [hunk 2]
```diff
@@ -272,7 +272,7 @@
                                 <!--<span class="premium"><i class="fa fa-lock" aria-hidden="true"></i> premium</span>-->
                                 <br />
                                 <small>
-                                    <i class="fa fa-download" aria-hidden="true"></i> <a href="https://pro.electricitymap.org/carts/new" target="_blank">download full dataset</a>
+                                    <i class="fa fa-download" aria-hidden="true"></i> <a href="https://pro.electricitymap.org/carts/new" target="_blank"><%= __('country-history.downloaddataset') %></a>
```
Comment (corradio) [inline web/views/pages/index.ejs]: 👍

File: .editorconfig (hunk 1 above)
Comment (magol) [inline .editorconfig]: @corradio ups, you are right. I do this in a separate PR. 
The reason that I change the indent for html to 2 was because https://github.com/tmrowco/electricitymap/blob/master/mobileapp/www/index.html use mostly 2 spaces as indent. But it is a small file, and it is best to use 4 spaces for all files.
